src/App.js

```diff
@@ -1,5 +1,37 @@
+import { MissionUtils } from '@woowacourse/mission-utils'
+import { MESSAGE } from './constants/message.js'
+import {
+	inputCarsNameAndValidate,
+	inputTryCountAndValidate,
+	printResult,
+	racingResult,
+	winner,
+} from './utils/index.js'
+
 class App {
-  async play() {}
+	async play() {
+		//자동차 이름 입력
+		MissionUtils.Console.print(MESSAGE.START)
+		const carsNameSplit = await inputCarsNameAndValidate()
+		let carsInfo = new Map(carsNameSplit.map(carName => [carName, 0]))
+
+		//시도할 횟수 입력
+		MissionUtils.Console.print(MESSAGE.TRY_COUNT)
+		const tryCount = await inputTryCountAndValidate()
+
+		//레이싱 결과 출력
+		MissionUtils.Console.print(MESSAGE.RESULT)
+
+		for (let i = 0; i < tryCount; i++) {
+			racingResult(carsInfo)
+			printResult(carsInfo)
+		}
+		const printWinner = winner(carsInfo)
+		MissionUtils.Console.print(`${MESSAGE.END}${printWinner}`)
```

Comment:
```suggestion
		MissionUtils.Console.print(MESSAGE.END + printWinner);
```
저는 개인적으로 템플릿 리터럴은 중간에 공백이나 다른 문자같은게 추가될 때 사용하는 것 같아요!
지금 상황에서는 추가되는게 없기 때문에 `+` 연산자를 사용하는게 가독성에 도움이 될 수 있을 것 같아요~!